.github/ISSUE_TEMPLATE/found-a-bug.yml

```diff
@@ -0,0 +1,51 @@
+name: 🐛 Found a Bug
+description: This template is for reporting bugs (broken or incorrect behaviour). If you have questions about your own code, please visit our forum discourse.processing.org instead.
+labels: [bug]
+body:
+- type: checkboxes
+  id: sub-area
+  attributes:
+    label: Most appropriate sub-area of p5.js?
+    description: You may select more than one.
+    options:
+      - label: Accessibility (Web Accessibility)
+      - label: Build tools and processes
+      - label: Color
+      - label: Core/Environment/Rendering
+      - label: Data
+      - label: DOM
+      - label: Events
+      - label: Friendly error system
+      - label: Image
+      - label: IO (Input/Output)
+      - label: Localization
+      - label: Math
+      - label: Unit Testing
+      - label: Typography
+      - label: Utilities
+      - label: WebGL
+      - label: Other (specify if possible)
+- type: textarea
+  attributes:
+    label: p5.js version 
+    description: You can find this in the first line of the p5.js file.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Web browser and version 
+    description: In the address bar, on Chrome enter "chrome://version", on Firefox enter "about:support". On Safari, use "About Safari".
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Operating System 
+    description: "Ex: Windows/MacOSX/Linux/Android/iOS along with version"
+  validations:
+    required: false
+- type: textarea
```

Comment:
Would be nice if it can be some kind of code field but not sure if that is possible at all.

Reply:
Hmm I don't think there is a code block field (although that is a good feat request for GitHub). However, I can add a default value to the field: 

<img width="548" alt="image" src="https://user-images.githubusercontent.com/17113462/143461698-2cbc9e6c-8b35-4fc9-b780-c6e9aa143011.png">

Which is then rendered as the following:

<img width="439" alt="image" src="https://user-images.githubusercontent.com/17113462/143461765-44c6568f-7124-4c9e-9f58-4fe58c25cb08.png">

What do you think? In terms of a friendly experience for first timers, I am not sure if having a default code block is better or no markdown at all is better.

Reply:
Is it able to add a blank line in the template shown to the user right above `### Snippet`? Just to make things a bit neater.

Reply:
I like the code block! maybe inside of the ticks it could say

```js
# Paste your code here :)
```

so if you get intimidated by the code block formatting, there's a clear invitation for where to put your example?